### PR TITLE
Lesson 12

### PR DESCRIPTION
**Pull request by namelexa**
### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/AKisilenko/ModuleLesson6/Controller/Adminhtml/Ask/MassStatus.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+     */
+    public function execute()
+    {
+        $statusValue = $this->getRequest()->getParam('status');
+        $selected = $this->getRequest()->getPostValue('selected');
+
+        $collection = $this->collectionFactory->create()->addFieldToFilter('ask_id', $selected);
+
+        foreach ($collection as $item) {
+            $item->setStatus($statusValue);
```
Comment (chabanenk0) [inline app/code/AKisilenko/ModuleLesson6/Controller/Adminhtml/Ask/MassStatus.php:57]: Try using of transaction object here

File: app/code/AKisilenko/ModuleLesson6/Model/AskQuestion.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    {
+
+        $this->_init(AskQuestionResource::class);
+    }
+
+//    /**
+//     * @return AbstractModel
+//     * @throws NoSuchEntityException
+//     */
+//    public function beforeSave()
```
Comment (chabanenk0) [inline app/code/AKisilenko/ModuleLesson6/Model/AskQuestion.php]: Why all this is commented? If it doesn't work, please ask for help, but seems it is needed

File: app/code/AKisilenko/ModuleLesson6/Controller/Adminhtml/Ask/MassStatus.php (hunk 1 above)
Comment (namelexa) [reply app/code/AKisilenko/ModuleLesson6/Controller/Adminhtml/Ask/MassStatus.php:57]: Transaction added

File: app/code/AKisilenko/ModuleLesson6/Model/AskQuestion.php (hunk 2 above)
Comment (namelexa) [reply app/code/AKisilenko/ModuleLesson6/Model/AskQuestion.php]: It works. I experimented with this, therefore commented out

**Pull request merged**